code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -356,6 +379,7 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/L, metabolis
 
 /datum/reagent/medicine/neuraline/on_mob_life(mob/living/L)
 	L.reagent_shock_modifier += PAIN_REDUCTION_FULL
+	L.adjustStaminaLoss(-30*REM)
```

Comment:
Could you explain the function and trade-offs of this chem?

Reply:
The function of this chem is a medic-only anti-stun chem.

The only way to acquire it is via an autoinjector in the medics vendor for... 10? points.
Each injector has 3 3u uses, and metabolizes at 1u/tick, while causing brain damage above 4u.
If you jab a downed guy who isn't in actual crit with this, they WILL get up.

The tradeoff is you can't make it or hack a vendor, you have to be/befriend a medic to get this, and it's pretty short duration.